src/containers/guest-home-page/styles/WelcomeBlockStyles.js

```diff
@@ -0,0 +1,104 @@
+import theme from '~/styles/app-theme/custom-mui.styles'
+import desktopBg from '~/img/guest-home-page/desktopBg.svg'
+import tabletBg from '~/img/guest-home-page/tabletBg.svg'
+import mobileBg from '~/img/guest-home-page/mobileBg.svg'
+import { blueGrey } from '@mui/material/colors'
+
+export const WelcomeBlockStyles = {
+  container: {
+    paddingTop: '144px',
+    paddingBottom: '206px',
+    backgroundImage: `url(${ desktopBg })`,
+    backgroundPosition: 'center -380px',
+    backgroundRepeat: 'no-repeat',
+    [theme.breakpoints.down('md')]: {
+      paddingTop: '72px',
+      paddingBottom: '80px',
+      backgroundImage: `url(${ tabletBg })`,
+      backgroundPosition: 'center -300px'
+    },
+    [theme.breakpoints.down('sm')]: {
+      paddingTop: '75px',
+      backgroundImage: `url(${ mobileBg })`,
+      backgroundPosition: 'center -260px'
+    }
+  },
+  title: {
+    'br:nth-of-type(1)': {
+      [theme.breakpoints.down('md')]: {
+        display: 'none'
+      }
+    },
+    'br:nth-of-type(2)': {
+      display: 'none',
+      [theme.breakpoints.down('md')]: {
+        display: 'flex'
+      },
+      [theme.breakpoints.down('sm')]: {
+        display: 'none'
```

Comment:
you can use breackpoints like this:
display: { xs: 'none', md: 'flex' }

Reply:
Fixed.